literature.bib

```diff
@@ -11483,6 +11585,19 @@ @article{BSRC10
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2009 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+% According to dl.acm.org the publisher is the Carnegie Mellon University and not ACM.
```

Comment:
According to IEEE, dblp and others (as well as the Proceedings front matter...) publisher is ACM

Reply:
But the other entries for SPLC09 also have publisher `SEI` which seems actually do be the dedicated variable I should have used here in the first place. One entry even has Springer listed as publisher but I have no idea why.

Reply:
There are conferences that switched their publisher multiple times.

Reply:
Yes, but for the same converence (SPLC 2009)? I changed the publisher now to `SEI` here,